django/contrib/admin/templates/admin/base.html

```diff
@@ -4,6 +4,7 @@
 <head>
 <title>{% block title %}{% endblock %}</title>
 <link rel="stylesheet" type="text/css" href="{% block stylesheet %}{% static "admin/css/base.css" %}{% endblock %}">
+<link rel="stylesheet" type="text/css" href="{% block dark-scheme %}{% static "admin/css/dark.css" %}{% endblock %}" media="(prefers-color-scheme: dark)">
```

Comment:
Nice media query on the link element.

Reply:
Yes, unfortunately it's not supported by [IE or Edge](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) :disappointed:

Reply:
It's OK. No-one uses those. 😀



('Twas a jest.)

Reply:
More seriously, support seems to be coming in Edge, and the whole point of these kind of queries is that they degrade harmlessly. A browser that doesn't understand `prefers-color-scheme` will just ignore it.

Reply:
But it means that all IE and Edge browsers will get a dark scheme. Am I right? :thinking:

Reply:
No, they won't download it.